cdap-runtime-spi/src/main/java/io/cdap/cdap/runtime/spi/launcher/LauncherFile.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.runtime.spi.launcher;
+
+import java.net.URI;
+import java.util.Objects;
+
+/**
+ * Represents file to be used while launching a job via {@link Launcher}.
```

Comment:
It's not clear to me what is in this file and how it should be used to launch a job.